UploadKitDemo/UploadKitDemo/UploadTaskDelegate.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UploadTaskProtocol.swift
+//  UploadKitDemo
+//
+//  Created by Huanming Hu  on 2016/10/29.
+//  Copyright © 2016年 UploadKit. All rights reserved.
+//
+
+import Foundation
+
+protocol UploadTaskDelegate {
```

Comment:
1. I think we need an interface of "process" to indicates how far the upload process goes.
var process(process: (Float)->Void) -> Self
2. I suggest we remove "_" in codes and simply use camel style.
var before(before: ()->Void) ->Self